modules/exploits/linux/http/fusionpbx_php_editor_cmd_exec.rb

```diff
@@ -0,0 +1,135 @@
+####################################################################
+# This module requires Metasploit: https://metasploit.com/download #
+#  Current source: https://github.com/rapid7/metasploit-framework  #
+####################################################################
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+        "Name" => "FusionPBX =< 4.5.10 Authenticated Remote Code Execution",
+        "Description" =>  %q{
+          Authenticated Remote Code Execution in FusionPBX web appliances.
+          Affected versions: =< 4.5.10
+          By using the on site PHP-Editor function users can overwrite PHP files on the web server
+          with a trojan horse and get a shell as www-data by navigating to the changed PHP file location.
+          Vendor verified: 11/27/2019
```

Comment:
The dates can be removed, theyre covered in the disclosure date field

Reply:
Ok, they will be gone in the next commit.